templates/site/design-component-preview.html

```diff
@@ -0,0 +1,126 @@
+{# TODO: This file is for testing components.  Remove this before merging. #}
```

Comment:
If this is ready to be merged, please remove

Reply:
Perhaps keep this as a preview page?  Move it to temp or something so the components can be viewed.

Reply:
This is useful for developers but shouldn't be visible on the live site. Please remove it from the `site` directory before this is merged.